app/models/drupal_users.rb

```diff
@@ -66,11 +66,11 @@ def like_count
   end
 
   def liked_notes
-    DrupalNode.includes(:node_selections).where("type = 'note' AND node_selections.liking = true AND node_selections.user_id = ? AND node.status = 1", self.uid).order('node_selections.nid DESC')
+    DrupalNode.includes(:node_selections).where("type = 'note' AND node_selections.liking = ? AND node_selections.user_id = ? AND node.status = 1", true, self.uid).order('node_selections.nid DESC')
```

Comment:
Had to make this change because the sqlite db was showing an error probably because it wasn't taking true as a boolean.